app/public/css/main.css

```diff
@@ -0,0 +1,1734 @@
+/**
```

Comment:
@juliangruber it's generated – i guess the entire `app` directory should be gitignored? (where does that even come from? i'm confused)